client/app/views/steps/password.coffee

```diff
@@ -13,17 +13,23 @@ module.exports = class PasswordView extends StepView
     isVisible: false
 
 
+    afterRender: ->
```

Comment:
`afterRender` does not exist in Backbone (and that's a shame), but Marionette provides an `onRender()` method triggered after each call to `render()`.
http://marionettejs.com/docs/v2.4.7/marionette.itemview.html#render--onrender-event